cmake_file_api/kinds/toolchains/v1.py

```diff
@@ -70,7 +70,11 @@ def __init__(self, language: str, compiler: CMakeToolchainCompiler, sourceFileEx
     def from_dict(cls, dikt: Dict) -> "CMakeToolchain":
         language = dikt["language"]
         compiler = CMakeToolchainCompiler.from_dict(dikt["compiler"])
-        sourceFileExtensions = dikt["sourceFileExtensions"]
+        if "sourceFileExtensions" in dikt:
+            sourceFileExtensions = dikt["sourceFileExtensions"]
+        else:
+            sourceFileExtensions = []
+
```

Comment:
I'm reading the spec at https://cmake.org/cmake/help/latest/manual/cmake-file-api.7.html#object-kind-toolchains, and it appears `soruceFileExtensions` is optional.
So please change it as follows:
```suggestion
        sourceFileExtensions = dikt.get("sourceFileExtensions")
```
Optional values are treated likewise in other places.

Reply:
thanks for the hint, I had overlooked that in the documentation - done